src/main/webapp/js/instructorFeedbackResults.js

```diff
@@ -209,15 +209,43 @@ function toggleCollapse(e, pans) {
     var expand = 'Expand';
     var collapse = 'Collapse';
     var panels = pans || $('div.panel-collapse');
+    var shouldExpandPanels = $(e).html().trim().startsWith(expand);
```

Comment:
What does `$(e).html().trim().startsWith(expand)` actually mean? panel currently collapsed? In that case the variable should be named something like `arePanelsCollapsed`?